docs/setup.md

```diff
@@ -0,0 +1,103 @@
+# Setup
+
+Hi, Welcome to the Mask Network. This guide will quickly take you through how to setup the extension development environment.
+
+## Requirements
+
+Here is a snippet of engines requirements in the package.json of Mask Network. As you see, NodeJS and pnpm is required at least a specific version.
+
+```json
+"engines": {
+    "node": ">=16.0.0",
+    "pnpm": ">=6.32.1",
```

Comment:
Suggest people use [corepack](https://nodejs.org/api/corepack.html) so they will no need to worry if their pnpm is the required version.

Our project is already configured with corepack.